plugins/processors/enum/enum.go

```diff
@@ -23,7 +23,9 @@ type EnumMapper struct {
 
 type Mapping struct {
 	Tag           string
+	Tags          []string
 	Field         string
+	Fields        []string
```

Comment:
Can we please add add TOML annotations here and deprecate the singular options in favor of the plural ones?